app/main/translate.py

```diff
@@ -59,16 +83,21 @@ def translate_from_to(source, target, text):
 
 
 #@logged()
-def split_to_sent_array(text, lang):
+def split_to_sent_array(text, lang, limit):
     sent_array = []
-    limit = current_app.config['SENT_LEN_LIMIT']
     for sent in split_text_into_sentences(text=text, language=lang):
         while len(sent) > limit:
             try:
-                last_space_idx = sent.rindex(" ", 0, limit)
+                # When sent starts with a space, then sent[0:0] was an empty string, 
+                # and it caused an infinite loop. This fixes it. 
```

Comment:
To je pravda jenom, kdyz prvni mezera je zaroven posledni. Ale good point. Otazkou spis je proc sentence splitter vraci vety zacinajici mezerou, to by imo delat nemel. Nemas nejak priklad?